src/shims/unix/linux/foreign_items.rs

```diff
@@ -156,22 +149,18 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                         this.write_scalar(Scalar::from_target_usize(len, this), dest)?;
                     }
                     // `futex` is used by some synchronization primitives.
-                    id if id == sys_futex => {
+                    op if op == sys_futex => {
                         futex(this, &args[1..], dest)?;
                     }
-                    id if id == sys_eventfd2 => {
-                        let [_, initval, flags, ..] = args else {
-                            throw_ub_format!(
-                                "incorrect number of arguments for `eventfd2` syscall: got {}, expected at least 3",
-                                args.len()
-                            );
-                        };
+                    op if op == sys_eventfd2 => {
+                        let [_, initval, flags] =
+                            check_min_arg_count("syscall(SYS_evetfd2, ...)", args)?;
 
                         let result = this.eventfd(initval, flags)?;
                         this.write_int(result.to_i32()?, dest)?;
                     }
-                    id => {
-                        throw_unsup_format!("can't execute syscall with ID {id}");
+                    op => {
+                        throw_unsup_format!("can't execute syscall with OP {op}");
```

Comment:
I switched to `op` to unify it with other places I have seen across the repo. Moreover, some `libc` variadic functions use `op` too.

Reply:
uppercase "OP" makes little sense though? ID is often used upper-case, but not op...

`man syscall` calls this "number", so maybe the best here is "can't execute syscall number {number}".

Reply:
If number suits you too, then let's change it. I'm just not sure what's intentional here and what naming do you prefer, so trying to be a bit more conservative (:

Reply:
Well the conservative thing would be to not change it. ;)

But I don't think "syscall id" or "syscall op" are common. "syscall number" however is common. So if we change it, it should be to "number"/`num`.